colors:

  red:
    H_range: [0, 17]
    S_range: [112, 255]
    V_range: [98, 255]

  green:
    H_range: [35, 80]
    S_range: [143, 255]
    V_range: [90, 255]

  blue:
    H_range: [95, 145]
    S_range: [143, 255]
    V_range: [90, 255]

  orange:
    H_range: [15, 35]
    S_range: [143, 255]
    V_range: [90, 255]

  white:
    H_range: [0, 180]
    S_range: [0, 58]
    V_range: [150, 255]

detection_config:
  min_area: 2.0
  poly_epsilon: 5.0
  kernel_type: 0
  kernel_size: 1
